lib/global_id/locator.rb

```diff
@@ -19,8 +19,11 @@ def locate(gid, options = {})
       end
 
       # Takes an array of GlobalIDs or strings that can be turned into a GlobalIDs.
-      # The GlobalIDs are located using Model.find(array_of_ids), so the models must respond to
-      # that finder signature.
+      # All GlobalIDs must belong to the same app, as they will be located using
+      # the same locator using its locate_many method.
+      #
+      # By default the GlobalIDs will be located using Model.find(array_of_ids), so the
```

Comment:
Ah, crap, Model.where was in the existing docs? I guess it's fine then :sweat_smile: